### DISCUSSION TITLE
注册成 Windows 服务如何添加环境变量

### DISCUSSION THREAD

**Discussion by boxiqiexi** [1715173329/UnblockNeteaseMusic#375] [Q&A] [answered]
是在nw.js中吗

**Comment by pan93412** [comment, marked as answer]
參考一下這個：<https://github.com/coreybutler/node-windows#environment-variables>

解決之後幫我按下「Mark as answer」，謝謝！

**Reply by boxiqiexi** [reply]
![MZ9(97MW@O61}S073(WKUYW](https://user-images.githubusercontent.com/55476331/145697308-e6125c4a-7de7-47aa-ab4a-84d056cfb283.png)
完美解决 谢谢

**Reply by pan93412** [reply]
> ![MZ9(97MW@O61}S073(WKUYW](https://user-images.githubusercontent.com/55476331/145697308-e6125c4a-7de7-47aa-ab4a-84d056cfb283.png)
> 完美解决 谢谢

其實正確應該是這樣：

```json
{
  "name": "ENABLE_FLAC",
  "value": "true"
}
```

你這樣會變成 `FLAC=true`

**Reply by boxiqiexi** [reply]
已更正
